### PR TITLE
Create Profile for Sam Wagner

### PR DESCRIPTION
**Pull request by SamBWagner**
Created Profile, no new badges

### CODE REVIEW

File: Sam-Wagner/Sam-Wagner.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+[[imgBadge]]
+| ![Linux](../badges/Developer-linux.png)
+
+[[imgBadge]]
+| ![NodeJS](../badges/Developer-node-js.png)
+
+[[imgBadge]]
+| ![SQL Server](../badges/Developer-sql-server.png)
+
+Sam is a Software Engineer at SSW. Coming form a background in low-level programming, but he has cut his teeth on web development using ASP.NET core, Entity Framework Core and Angular. He's also got experience with the Power platform using PowerApps, PowerAutomate and Microsoft's Dataverse to build CRMs.
```
Comment (bradystroud) [inline Sam-Wagner/Sam-Wagner.md:43]:
```suggestion
Sam is a Software Engineer at SSW. Coming from a background in low-level programming, but he has cut his teeth on web development using ASP.NET core, EF Core and Angular. He's also got experience with the Power Platform using PowerApps, PowerAutomate and Microsoft's Dataverse to build CRMs.
```

File: Sam-Wagner/Sam-Wagner.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+## .NET
+
+Sam's loves building WebAPIs with ASP.NET Core, working to build the fastest, safest and most reliable backends he can.
+
+## Angular
+
+Front-End web development is a subtle love of Sam's. A sneaky designer at heart, he loves all the trials and tribulations that come with building a beautiful front-end.
```
Comment (bradystroud) [inline Sam-Wagner/Sam-Wagner.md:51]:
```suggestion
Front-end web development is a subtle love of Sam's. A sneaky designer at heart 🦝, he loves all the trials and tribulations that come with building a beautiful front-end.
```

**Review by bradystroud — CHANGES_REQUESTED** [review]
Looking good 🙂
...A few changes

**Comment by SamBWagner** [comment]
Like the changes brother man! 👏

**Review by SamBWagner — COMMENTED** [review]
Looks Great!

**Review by bradystroud — APPROVED** [review]

**Pull request merged**